Take into account the encryption setting

marc-aurele-besner opened the pull request:
## Take into account the encryption setting

Regardless of DSN_ENCRYPT_UPLOADS=false, if DSN_ENCRYPTION_PASSWORD has a password (and by default it does) it will encrypt the metadata

jfrank-summit commented:
Would it just make more sense to remove DSN_ENCRYPT_UPLOADS from the config as it is redundant?

marc-aurele-besner commented:
> Would it just make more sense to remove DSN_ENCRYPT_UPLOADS from the config as it is redundant?

> Would it just make more sense to remove DSN_ENCRYPT_UPLOADS from the config as it is redundant?

I changed the solution and removed the default password
